CHANGELOG.md

```diff
@@ -42,6 +41,8 @@ The types of changes are:
 - Added custom JSON (de)serialization to shared DB engines to handle non-standard data types in JSONB columns [#4818](https://github.com/ethyca/fides/pull/4818)
 - Added state persistence across sessions to the datamap report table [#4853](https://github.com/ethyca/fides/pull/4853)
 - Removed currentprivacypreference and lastservednotice tables [#4846](https://github.com/ethyca/fides/pull/4846)
+- Added initial version for Helios: Data Discovery and Detection [#4839](https://github.com/ethyca/fides/pull/4839)
+- Adds new var to track fides js overlay types [#4869](https://github.com/ethyca/fides/pull/4869)
```

Comment:
some unrelated changelog maintenance while i was here: these were released in 2.36.0 and showed up under 2.36.0 on that release branch/tag, so just adjusting in `main` to keep that correct going forward